src/components/utils/include/utils/file_system.h

```diff
@@ -70,10 +70,10 @@ uint64_t FileSize(const std::string& path);
 
 /**
  * @brief Creates directory with owner_all permissions
- * @param name path to directory
- * @return path to created directory.
+ * @param path directory to create
+ * @return return true if directory was created or already exist.
```

Comment:
@SKobziev duplicate

Reply:
This description has the same structure as other descriptions of methods in this class. For example, you can look at the descriptions of CreateDirectoryRecursively, IsDirectory and etc.

Reply:
@SKobziev You have added duplicate... It was better before changes

Reply:
@AByzhynar I have corrected it yet. Could you please check?